### PR TITLE
November monthly update

### PR DESCRIPTION
**Pull request by xdamman**

### CODE REVIEW

**Comment by piamancini** [comment]
nice!

File: server/lib/emailTemplates.js [hunk 1]
```diff
@@ -62,6 +62,7 @@ handlebars.registerHelper('moment', (value, props) => {
 
 handlebars.registerHelper('currency', (value, props) => {
   const { currency, precision } = props.hash;
+  if (!currency) return value;
```
Comment (asood123) [inline server/lib/emailTemplates.js]: What's the scenario where we send a value without currency? Just like last time, it's going to send out emails without currency attached.

File: templates/emails/group.monthlyreport.hbs [hunk 2]
```diff
@@ -28,43 +34,93 @@ ul li {
...
 
 <div style="font-size:10pt">
 
 <h1>{{group.name}} {{month}} report</h1>
 
-<p>Hi there! This is the monthly report for your open collective with your latest balance (as of the last day of last month) and all the expenses that were submitted.
-You can always visit your collective page to get the latest up to date data: <a href="{{group.publicUrl}}">{{group.publicUrl}}</a><br />
-Don't hesitate to send us your feedback by replying to this email, we thrive on feedback! ❤️ ❤️</p>
+<p>Hi!<br />
+We have so much great news this month, we could barely wait to share them with you! Please remember, don't hesitate to send us your feedback by replying to this email, we thrive on feedback! ❤️ ❤️</p>
```
Comment (asood123) [inline templates/emails/group.monthlyreport.hbs]: "so much great news" doesn't sound right. "We have lots of good news" is better.

Comment (asood123) [inline templates/emails/group.monthlyreport.hbs]: Second sentence "Please don't hesitate to send us feedback - just reply to this email! ❤️ ❤️</p>"

File: templates/emails/group.monthlyreport.hbs [hunk 3]
```diff
@@ -28,43 +34,93 @@ ul li {
...
+But first, your monthly stats (you can always visit your collective page to get the latest up to date data: <a href="{{group.publicUrl}}">{{group.publicUrl}}</a></p>
```
Comment (asood123) [inline templates/emails/group.monthlyreport.hbs]: Missing close parenthesis

File: templates/emails/group.monthlyreport.hbs [hunk 4]
```diff
@@ -100,44 +156,39 @@ Don't hesitate to send us your feedback by replying to this email, we thrive on
...
+<p> </p>
+<p><a href="https://community.algolia.com/">Algolia</a> is grateful for the work that you do and wants to invite all of you to try out their<a href="https://community.algolia.com/docsearch/"> </a><a href="https://community.algolia.com/docsearch/">DocSearch project</a> for adding a really 👌 search to your documentation. Better search means fewer duplicate issues or support tickets from users \o/. It's free for Open Source Software and Algolia will do 90% of the work to get you up and running,<a href="https://community.algolia.com/docsearch/"> </a><a href="https://community.algolia.com/docsearch/">just fill out this form</a> to get started. If you have any questions or need support (about anything search-related, really!),<a href="https://discourse.algolia.com"> </a><a href="https://discourse.algolia.com">log on to their Discourse forum</a> and post in the DocSearch category with the #opencollective tag. Read more about why they do this <a href="https://stories.algolia.com/why-were-backing-opencollective-and-webpack-39e39838f18a#.iagl7sfu1">here</a>! </p>
```
Comment (asood123) [inline templates/emails/group.monthlyreport.hbs]: Several extra <a> tags without anything in the middle of it.

Comment (asood123) [inline templates/emails/group.monthlyreport.hbs]: ```<a href="https://community.algolia.com/docsearch/"> </a>```
```<a href="https://discourse.algolia.com"> </a>
```

File: templates/emails/group.monthlyreport.hbs [hunk 5]
```diff
@@ -100,44 +156,39 @@ Don't hesitate to send us your feedback by replying to this email, we thrive on
...
-<li><a href="https://medium.com/open-collective/asking-for-a-new-feature-on-an-open-source-project-3305deba1356#.vbccjusi0">Funding for Server-side rendering in React-boilerplate</a>: Help us raise money to fund the development of <a href="https://github.com/mxstbr/react-boilerplate/issues/1103">Server-Side Rendering in React-boilerplate</a>.<br />
-- <a href="https://opencollective.com/react-boilerplate/donate/50">Donate $50</a><br />
-- <a href="https://opencollective.com/react-boilerplate/donate/100">Donate $100</a><br />
-- <a href="https://opencollective.com/react-boilerplate/donate/250">Donate $250</a> (like we did :) ).</li>
-</ul>
+<p>Together we crowdfunded a yearly budget of over $100,000 for Open Source (and we are only getting started!). Help us share this <a href="https://medium.com/open-collective/together-we-crowdfunded-a-yearly-budget-of-over-100-000-for-open-source-a77502fd74da#.y902opkkp">blog post</a>! </p>
```
Comment (asood123) [inline templates/emails/group.monthlyreport.hbs]: s/yearly/annual

File: server/lib/queries.js [hunk 6]
```diff
@@ -179,23 +179,20 @@ const getTopSponsors = () => {
...
-        SUM(amount) as amount
-      FROM "Donations" d
-      WHERE d."GroupId" IN (:groupids) AND d.amount >= 0
-      GROUP BY "UserId"
-    ), last_donation AS (
+    WITH stats AS (
       SELECT
         max("UserId") as "UserId",
-        max("updatedAt") as "updatedAt"
+        SUM("amountInTxnCurrency") as "totalDonations",
```
Comment (asood123) [inline server/lib/queries.js]: This will send the results in cents, instead of dollars. Are you dividing by a 100 somewhere else?

**Pull request merged**